Metabase Functions.R

```diff
@@ -152,8 +152,7 @@ table_output <- NULL
   #for (i in 1:length(gender_table)) {
     
     demographics_table<-var_by_Org[[i]]
-    print(levels(Org)[i])
-    table_output[[i]]<-gt(demographics_table) %>% tab_header(title=paste(names(demographics_table)[1]," in ",levels(Org)[i]))%>% 
+    table_output[[i]]<-gt(demographics_table) %>% tab_header(title=paste(names(demographics_table)[1]," in ",i))%>% 
```

Comment:
On line 145, we run `for (i in levels(by))`.
This means that we are setting `i` to equal a level in the `Org` variable (since `by = Org`). 
For example, `i = "Nontobeko"`.
As a result, to call the organisation we are grouping by, we just need to call `i` here, and not anything else.

The issue we had earlier is that I thought `i` was a numerical value, but it turns out it is already the factor level in the `by` variable (`Org`).

E.g.

```
# we are running this (effectively):
for (i in levels(plhdata_org_clean$Org)){
  print(i)
}
#[1] "Other"
#[1] "Nontobeko"
#[1] "Joy"
#[1] "Amathuba"
#[1] "Dlalanathi"

# I thought we were running this:
for (i in 1:length(levels(plhdata_org_clean$Org))){
  print(i)
}
#[1] 1
#[1] 2
#[1] 3
#[1] 4
#[1] 5
```

Reply:
Really useful summary, thanks!